student.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'person'
+class Student
+  def initialize(classroom, age, name = 'unknown', parent_permission: true)
+    super(name, age, parent_permission: parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯\(ツ)/¯'
+  end
+end
```

Comment:
- According to the project requirements, please ensure that your `Student` class inherits from the `Person` class. This [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#inheritance) will help you understand better. Please fix this 🙏